lxd/device/device_utils_disk.go

```diff
@@ -401,7 +402,10 @@ func DiskVMVirtfsProxyStop(pidPath string) error {
 		}
 
 		// Remove PID file.
-		os.Remove(pidPath)
+		err = os.Remove(pidPath)
+		if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
I think we can actually get away with just ignoring the error here as the command will fail to start if it cant write its new pid file anyway.